server/src/test/java/keywhiz/service/daos/SecretSeriesDAOTest.java

```diff
@@ -566,5 +649,28 @@ private long createSecretContent(long secretId) {
   private static String randomName() {
     return UUID.randomUUID().toString();
   }
+
+  private static long randomExpiration() {
+    long expiration = 0;
+    Random random = new Random();
+
+    for (;;) {
+      expiration = random.nextLong();
+
+      if (expiration <= 0) {
+        continue;
+      }
+
+      try {
+        Instant.ofEpochSecond(expiration);
+      } catch (DateTimeException e) {
+        continue;
+      }
+
+      break;
+    }
+
+    return expiration;
+  }
```

Comment:
Because the Keywhiz APIs bounce back and forth between long and Instant, we have to ensure that the expiration we generate is valid for both (this came up during a test where the random long value was outside the bounds of what's allowed for Instant seconds).